bootstrap/terraform/aws-bootstrap/locals.tf

```diff
@@ -0,0 +1,13 @@
+locals {
+    sts_principal = "sts.${data.aws_partition.current.dns_suffix}"
+
+    private_subnet_ids = var.create_cluster ? module.vpc.private_subnets_ids : var.private_subnet_ids
+    public_subnet_ids = var.create_cluster ? module.vpc.public_subnets_ids : var.public_subnet_ids
```

Comment:
Do we need these variables if we can get them from the cluster data object already?